src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -430,6 +455,27 @@ impl A320Hydraulic {
         self.yellow_loop.is_pressurised()
     }
 
+    #[allow(clippy::too_many_arguments)]
+    // Update with same refresh rate as the sim
+    fn update_max_fixed_step(&mut self, context: &UpdateContext) {
+        while let Some(cur_time_step) = self.physics_updater.next() {
+            self.forward_cargo_door.update(
+                &self.forward_cargo_door_controller,
+                &context.with_delta(cur_time_step),
+                self.yellow_loop.pressure(),
+            );
+
+            self.aft_cargo_door.update(
+                &self.aft_cargo_door_controller,
+                &context.with_delta(cur_time_step),
+                self.yellow_loop.pressure(),
+            );
+
+            self.ram_air_turbine
+                .update_physics(&cur_time_step, context.indicated_airspeed());
+        }
+    }
+
     #[allow(clippy::too_many_arguments)]
     // Update with same refresh rate as the sim
```

Comment:
**thought**: The comment on `update_every_frame` is equal to the comment on `update_max_fixed_step`, yet they do seem to update at different rates. Something feels wrong with this.